src/components/DataManager/Toolbar/Toolbar.js

```diff
@@ -22,7 +23,7 @@ export const Toolbar = injector(observer(({ store }) => {
               return Instrument ? (
                 <Instrument
                   key={`instrument-${instrument}-${i}`}
-                  size="medium"
+                  size={isFF(FF_LOPS_E_3) ? "large" : "medium"}
```

Comment:
Please update the label and review buttons on DataManager to use the correct size.

Reply:
done 
<img width="1177" alt="image" src="https://user-images.githubusercontent.com/104568407/229116552-2b5e991d-aff0-4ef2-8017-c845aed89027.png">